sdk/core/azure-core/azure/core/tracing/abstract_span.py

```diff
@@ -1,4 +1,14 @@
-from abc import ABC, abstractmethod, abstractproperty, abstractstaticmethod
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import abc
+from abc import abstractmethod
+
+try:
+    ABC = abc.ABC
+except AttributeError:  # Python 2.7, abc exists, but not ABC
+    ABC = abc.ABCMeta("ABC", (object,), {"__slots__": ()})  # type: ignore
```

Comment:
I guess we are trying to avoid `six` in azure-core? We've already taken the dependency elsewhere FWIW

Reply:
Hmm. I copied this from the pipeline.
https://github.com/Azure/azure-sdk-for-python/blob/15e5ff55dc9fd4e095d9f675f10378e952b4c3e3/sdk/core/azure-core/azure/core/pipeline/__init__.py#L30-L33
Should I change it to use `@six.add_metaclass(abc.ABCMeta)`?

Reply:
Probably not, if that is already established elsewhere.

Reply:
ok cool